csclient/csclient_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
-	"net/url"
+	neturl "net/url"
```

Comment:
why change this when you don't appear to be using the url name for anything else anyway?